src/main/java/net/datafaker/providers/base/Planet.java

```diff
@@ -0,0 +1,75 @@
+package net.datafaker.providers.base;
+
+import java.util.Locale;
+
+/**
+ * Provides planet specific metadata like length of the day, radius, mass etc.
+ *
+ * @since 2.1.1
```

Comment:
I think 2.2.0 will be a more likely candidate, 2.1.1 would imply a bigfix

Reply:
? Readme currently show 2.0.3 snapshot, so isn’t 2.0.3 the next version?

Reply:
The release is only half done due to an issue at Sonatype, for some reason the 2.1.0 couldn't get published, but it seems like they resolved it now.

Reply:
And for adding new functionality, 2.2.0 would make sense from a semver perspective (not that I'm 100% strict with that, but when x.y.z, I like to reserve Y for new functionality, and Z for bugfixes.

Reply:
Okay fair.

Do you have a script that does the version updates in readme etc or do you normally do that manually?

Reply:
I type it in as part of mvn release. It's still the final part which needs to be automated, but I am not sure if we could get it smart enough to know the difference between a feature release and a bugfix release.

Reply:
Folks, if it would be up to me then 2.2.0 suits the best I think :) The next major version all in all.

I'll adjust the code upon the next modification.